.yamllint

```diff
@@ -0,0 +1,21 @@
+extends: default
+rules:
+  line-length:
+    max: 160
+  document-start: disable
+  indentation:
+    indent-sequences: true
```

Comment:
I think we should also agree on the general indentation:
```
    spaces: 2
```

Reply:
added in [c82e558](https://github.com/os-autoinst/os-autoinst-common/commit/c82e5584184650236fea313e69ba09f4031e2476)